CHANGELOG.md

```diff
@@ -0,0 +1,81 @@
+<a name="2.0.0"></a>
+# 2.0.0 (2017-11-20)
+
+* chore(ci): updating Continuous Integration ([591ca1a](https://github.com/mongodb-js/mongodb-extjson/commit/591ca1a))
+* chore(comments): removing commented out code ([967188c](https://github.com/mongodb-js/mongodb-extjson/commit/967188c))
+* chore(package.json): fixing errors in package.json ([b058377](https://github.com/mongodb-js/mongodb-extjson/commit/b058377))
+* chore(yarn): updating lockfile ([ea67aa8](https://github.com/mongodb-js/mongodb-extjson/commit/ea67aa8))
+* test(web): removed web test cases, and updated for 2.0 ext-json spec ([271aadd](https://github.com/mongodb-js/mongodb-extjson/commit/271aadd))
+* feat(*): make compliant with extended JSON 2.0.0 spec and add corpus tests ([7818d61](https://github.com/mongodb-js/mongodb-extjson/commit/7818d61))
+* feat(binary): removing browser buffer support ([dc556c8](https://github.com/mongodb-js/mongodb-extjson/commit/dc556c8))
+* style(*): format and lint entire codebase ([56992f5](https://github.com/mongodb-js/mongodb-extjson/commit/56992f5))
+
+
+### BREAKING CHANGE
+
```

Comment:
breaking changes need to include a description of how you no longer "new up" an extJSON variable.  It used to be like this:
```
const extJSON = require('mongodb-extjson');
const EJSON = new extJSON();
EJSON.parse(...);
```
now it is:
```
const EJSON = require('mongodb-extjson');
EJSON.parse(...);
```